scripts/install-toolchain.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+PLATFORM=$(uname -s | tr '[:upper:]' '[:lower:]')
+ARCH=$([[ $(uname -m) = "x86_64" ]] && echo 'amd64' || echo 'arm64')
+
+mkdir -p "${CHART_TMP_DIR}"
+
+## Install kubeconform
+mkdir -p "${CHART_TMP_DIR}/kubeconform"
+curl -sSL https://github.com/yannh/kubeconform/releases/download/${KUBECONFORM_VERSION}/kubeconform-${PLATFORM}-${ARCH}.tar.gz | tar xz -C "${CHART_TMP_DIR}/kubeconform"
+mv "${CHART_TMP_DIR}/kubeconform/kubeconform" "${CHART_TOOLS_DIR}/kubeconform"
+
+## Install helm v3
+mkdir -p "${CHART_TMP_DIR}/helmv3"
+curl -sSL https://get.helm.sh/helm-${HELMV3_VERSION}-${PLATFORM}-${ARCH}.tar.gz | tar xz -C "${CHART_TMP_DIR}/helmv3"
+mv "${CHART_TMP_DIR}/helmv3/${PLATFORM}-${ARCH}/helm" "${CHART_TOOLS_DIR}/helmv3"
+
+rm -rf ${CHART_TMP_DIR}
```

Comment:
```suggestion
rm -rf "${CHART_TMP_DIR}"
```
There are a couple places in this script and others where you might want to quote to prevent globbing/splitting.

Reply:
Should all be fixed!